alacritty/src/event.rs

```diff
@@ -568,6 +582,13 @@ impl<N: Notify + OnResize> Processor<N> {
                     processor.ctx.display_update_pending.message_buffer = true;
                     processor.ctx.terminal.dirty = true;
                 },
+                Event::ClipboardStore(clipboard_type, content) => {
+                    processor.ctx.clipboard.store(clipboard_type, content);
+                },
+                Event::ClipboardLoad(clipboard_type, transform) => {
```

Comment:
```suggestion
                Event::ClipboardLoad(clipboard_type, format) => {
```

I think formatting is a bit better here than just 'transform', just because it's a little less generic.